internal/task/executor.go

```diff
@@ -68,5 +70,8 @@ func runTaskSafely(ctx context.Context, t Task, resolver file.Resolver, s sbomsy
 		}
 	}()
 
-	return t.Execute(ctx, resolver, s)
+	start := time.Now()
+	res := t.Execute(ctx, resolver, s)
+	log.Tracef("task name: %s time to execute: %v", t.Name(), time.Since(start))
```

Comment:
I should have mentioned this on the first review we should lean into separating out data fields from the message itself:
```go
log.WithFields("task", t.Name(), "elapsed", time.Since(start))
```

Reply:
fixed